public/apps/configuration/panels/role-list.tsx

```diff
@@ -45,38 +46,38 @@ import { renderCustomization, truncatedListView } from '../utils/display-utils';
 
 const columns = [
   {
-    field: 'role_name',
+    field: 'roleName',
     name: 'Role',
     render: (text: string) => (
       <a href={buildHashUrl(ResourceType.roles, Action.view, text)}>{text}</a>
     ),
     sortable: true,
   },
   {
-    field: 'cluster_permissions',
+    field: 'clusterPermissions',
     name: 'Cluster permissions',
     render: truncatedListView(),
     truncateText: true,
   },
   {
-    field: 'index_permissions',
-    name: 'Index patterns',
+    field: 'indexPermissions',
+    name: 'Index permissions',
     render: truncatedListView(),
     truncateText: true,
   },
   {
-    field: 'internal_users',
-    name: 'Internal Users',
+    field: 'internalUsers',
+    name: 'Internal users',
     render: truncatedListView(),
   },
   {
-    field: 'backend_roles',
-    name: 'Backend Roles',
+    field: 'backendRoles',
```

Comment:
nonblocking: open for discussion, the name "backend role" is indeed confusing. Shall we just change every place to be external identities? Only leave the raw data transformation to have backend role.

Reply:
It is a good idea. I would like to do in a separate PR.